apps/ui/src/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { ConfigProvider, Flex, Image, Layout, Typography } from 'antd';
+
+import { pcglFooterTheme } from '@/components/providers/ThemeProvider';
+
+import PCGLFOOTER from '@/assets/pcgl-logo-footer.png';
```

Comment:
These two can go together, since they're both `@/something` (same type of import, "internal").
```suggestion
import PCGLFOOTER from '@/assets/pcgl-logo-footer.png';
import { pcglFooterTheme } from '@/components/providers/ThemeProvider';
```

Reply:
So instead of having multiple **@/something/*** just consolidate it into one alias to represent the path from src?
So something like this **@/*** ?

Reply:
nvm, I understand what you were trying to say after reading it again XD